modules/backend/widgets/Filter.php

```diff
@@ -39,7 +42,7 @@ class Filter extends WidgetBase
     //
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
```

Comment:
The `{}` have been stripped for a reason, the [API docs](https://octobercms.com/docs/api/) don't support them.